spec/index.bs

```diff
@@ -1958,12 +1958,20 @@ interface GPUTextureUsage {
 
             **Arguments:**
             <pre class=argumentdef for="GPUDevice/createTexture(descriptor)">
-                descriptor: Description of the {{GPUTexture}} to create.
+                |descriptor|: Description of the {{GPUTexture}} to create.
             </pre>
 
             **Returns:** {{GPUTexture}}
 
-            Issue: Describe {{GPUDevice/createTexture()}} algorithm steps.
+            1. If any of the following conditions are unsatisfied, generate a validation
+                error and return an [=invalid=] {{GPUTexture}}.
+                <div class=validusage>
+                    - |size.x| is greater than or equal to `1`.
```

Comment:
We don't have a variable named "size.x" defined in scope. Check out how this is specified in other places, like `createBuffer`. This would be `|descriptor|.{{GPUTextureDescriptor/size}}.width`